README.md

```diff
@@ -0,0 +1,69 @@
+Assisted Injection for JSR 330.
+============
+
+Manually injected dependencies for your JSR 330 configuration.
```

Comment:
Maybe link to https://github.com/google/guice/wiki/AssistedInject or just straight-up copy a bunch of their content?